Azure/azure-cli
# [ACR] Support disable public network access

This is pending SDK release

**Description**  
Support  `--public-network-access-enabled`
Improve tab output of `az acr show-endpoints`

**Testing Guide**  

az acr update/create --public-network-access-enabled

CC @djyou 

This checklist is used to make sure that common guidelines for a pull request are followed.
